.gitignore

```diff
@@ -9,6 +9,10 @@ bytecode.bin
 gobuild
 network.d*
 network.js*
+hardhat.config.d*
+hardhat.config.js*
+scripts/*.d*
+scripts/*.js*
```

Comment:
The change starts transpiling the scripts and hardhat config typescript files and publishing them so consumers can use npm to get them.